invoker/core/src/test/java/com/google/cloud/functions/invoker/IntegrationTest.java

```diff
@@ -252,6 +253,34 @@ public void helloWorld() throws Exception {
             ROBOTS_TXT_TEST_CASE));
   }
 
+  @Test
```

Comment:
I think it's okay that this has just an integration test, since the whole purpose of the timeoutfilter is to deal with time.  There's fancy google/guava libraries for mocking out the time and timer components but i don't think it's worth messing with those, because the scope here is pretty clear and is unlikely to change.